### PR TITLE
Support cover mode in pointcloud segment

### PR DESCRIPTION
**Pull request by MMMMoriaty**

### CODE REVIEW

File: packages/lb-utils/src/i18n/resources.json [hunk 1]
```diff
@@ -419,6 +419,8 @@
     "CameraFollowTopView": "按俯视图定位",
     "MarkOnlyOne": "仅标注一帧数据，需标出首尾的帧数据",
     "HaveNoNeed": "两帧之间暂无需预测的数据，请先标出首尾的框，需同序号",
-    "PredictingDataFrom": "正在预测第 {{startPage}} 页到第 {{endPage}} 页之间框ID为 {{selectedBoxTrackID}} 的数据"
+    "PredictingDataFrom": "正在预测第 {{startPage}} 页到第 {{endPage}} 页之间框ID为 {{selectedBoxTrackID}} 的数据",
+    "PointCloudCoverMode": "选中相同点云时，后一标记覆盖前一标记",
+    "PointCloudUncoverMode": "选中相同点云时，后一标记不可覆盖前一标记",
```
Comment (Kerwin-L) [inline packages/lb-utils/src/i18n/resources.json:424]: You need to add En-version.

Comment (MMMMoriaty) [reply packages/lb-utils/src/i18n/resources.json]: done

File: packages/lb-annotation/src/core/pointCloud/store/index.ts [hunk 2]
```diff
@@ -273,13 +295,37 @@ class PointCloudStore {
...
+
+        // update points in threeJs
+        const originPoints = this.scene.getObjectByName(id) as THREE.Points;
+        if (originPoints) {
+          originPoints.geometry.setAttribute('position', new THREE.Float32BufferAttribute(newPoints, 3));
+          originPoints.geometry.attributes.position.needsUpdate = true;
+        }
+      }
+    });
+    this.renderer.render(this.scene, this.camera);
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/core/pointCloud/store/index.ts:326]: Store doesn't directly execute redner. You can use `this.emit('reRender3d');`

File: packages/lb-annotation/src/core/pointCloud/render/index.ts [hunk 3]
```diff
@@ -99,32 +105,48 @@ class PointCloudRender {
...
     newPoints.name = segmentData.id;
 
     this.store.scene.add(newPoints);
     this.render3d();
   }
 
-  public updateNewPoints = (segmentData = this.store.cacheSegData) => {
+  public updateNewPoints = (segmentData: IPointCloudSegmentation) => {
     const originPoints = this.store.scene.getObjectByName(segmentData?.id ?? '') as THREE.Points;
+    const renderPoints =
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/core/pointCloud/render/index.ts:126]: logic needs to be removed from the render. Render just get the data to draw

File: packages/lb-annotation/src/core/pointCloud/store/index.ts [hunk 4]
```diff
@@ -191,18 +206,23 @@ class PointCloudStore {
...
-              this.cloudData.get(key).visible = false;
+          if (this.segmentMode === EPointCloudSegmentMode.Remove) {
+            this.cloudData.get(key).visible = false;
+            vertices.push(cloudDataArrayLike[i], cloudDataArrayLike[i + 1], cloudDataArrayLike[i + 2]);
+          }
+          if (this.segmentMode === EPointCloudSegmentMode.Add) {
+            if (this.cloudData.get(key).visible === true) {
+              covers.push(cloudDataArrayLike[i], cloudDataArrayLike[i + 1], cloudDataArrayLike[i + 2]);
             }
             vertices.push(cloudDataArrayLike[i], cloudDataArrayLike[i + 1], cloudDataArrayLike[i + 2]);
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/core/pointCloud/store/index.ts:217]: You can add your coverMode's judgement in this place

File: packages/lb-annotation/src/core/pointCloud/store/index.ts (hunk 2 above)
Comment (MMMMoriaty) [reply packages/lb-annotation/src/core/pointCloud/store/index.ts]: done

File: packages/lb-annotation/src/core/pointCloud/render/index.ts (hunk 3 above)
Comment (MMMMoriaty) [reply packages/lb-annotation/src/core/pointCloud/render/index.ts:126]: done

File: packages/lb-annotation/src/core/pointCloud/store/index.ts (hunk 4 above)
Comment (MMMMoriaty) [reply packages/lb-annotation/src/core/pointCloud/store/index.ts:217]: done

**Pull request merged**